src/main/java/com/chrosciu/bootcamp/tasks/github/GithubClient.java

```diff
@@ -4,28 +4,29 @@
 import com.chrosciu.bootcamp.tasks.github.dto.Repository;
 import lombok.RequiredArgsConstructor;
 import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
 
 @RequiredArgsConstructor
 public class GithubClient {
     private final GithubApi githubApi;
 
     public Flux<Repository> getUserRepositories(String username) {
-        //TODO: Implement
-        return null;
+        return githubApi.getUserRepositories(username).flatMapMany(Flux::fromIterable).onErrorReturn(new Repository());
```

Comment:
Nie do końca dobre podejście - zwrócenie pustego repozytorium (nie zawierającego żadnych danych) to nie jest dobry sposób na wybrnięcie z błędu - ktoś kto będzie potem ten strumień przetwarzał to może się przejechać np. na nullowej nazwie takiego repo. Jeżeli już iść w tę stronę to należałoby użyć onErrorResume i następnie zwrócić pusty strumień.